src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/ConfigModel/ParameterSymbol.cs

```diff
@@ -251,7 +251,7 @@ private static TemplateParameterPrecedence GetPrecedence(bool isRequired, bool i
                 return null;
             }
 
-            // Attribute parseable as a bool - so we do not want to present it as a condition
+            // Attribute parse-able as a bool - so we do not want to present it as a condition
```

Comment:
This should be parsable. (Parseable is technically correct, but relates more to breaking something down I think)